### PR TITLE
Fix for Issue #580

### PR DESCRIPTION
**Pull request by keflavich**
Everything broke, this is to try to get tests green again

### CODE REVIEW

**Comment by keflavich** [comment]
#580

**Comment by keflavich** [comment]
@astrofrog re: https://github.com/astropy/reproject/pull/166#issuecomment-535939570 see https://github.com/radio-astro-tools/spectral-cube/pull/581/commits/629860a77a66b0a2392a44a0fae513041574b781

**Comment by keflavich** [comment]
The remaining issues are some interaction w/numpy 1.17 and astropy

**Comment by keflavich** [comment]
@e-koch this should fix the outstanding issues that were holding back your other PRs, so with your approval, let's merge this and then get your others merged.

File: spectral_cube/spectral_cube.py [hunk 1]
```diff
@@ -2563,25 +2567,30 @@ def reproject(self, header, order='bilinear', use_memmap=False,
         shape_out = tuple([header['NAXIS{0}'.format(i + 1)] for i in
                            range(header['NAXIS'])][::-1])
 
+        if filled:
+            data = self.filled_data[:]
+        else:
+            data = self._data
```
Comment (e-koch) [inline spectral_cube/spectral_cube.py]: Is there a unit difference between `_data` and `filled_data`? Does it matter if the output is written to a memmap? This hasn't been a issue and is the default setting so maybe it doesn't matter.

Comment (keflavich) [reply spectral_cube/spectral_cube.py]: huh, should probbably be `unitless_filled_data` here....

Comment (keflavich) [reply spectral_cube/spectral_cube.py]: the units should be the same & inherited from the cube

**Pull request merged**